src/components/buttons/ButtonWithIcon.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Button, Icon, Col } from 'patternfly-react';
+import './ButtonWithIcon.css';
+
+export const ButtonWithIcon = (props) => {
+  return (
+    <Button onClick={props.onClick} bsClass="btn btn-link btn-with-icon">
+      <Col md={12}>
+        <Icon type={props.iconType} name={props.icon} className="fa-5x"/>
+      </Col>
+      <Col md={12} className="label-column">
+        <span className="lead">{props.label}</span>
+      </Col>
+    </Button>
+  );
+}
```

Comment:
1. please use extract the used props, imho it increases readability, you can use lebab to do a lot of this for you.
2. please add prop-types

the output should look something like
```js
import React from 'react';
import PropTypes from 'prop-types';
import { Button, Icon, Col } from 'patternfly-react';
import './ButtonWithIcon.css';

export const ButtonWithIcon = ({ onClick, iconType, icon, label }) => (
  <Button onClick={onClick} bsClass="btn btn-link btn-with-icon">
    <Col md={12}>
      <Icon type={iconType} name={icon} className="fa-5x" />
    </Col>
    <Col md={12} className="label-column">
      <span className="lead">{label}</span>
    </Col>
  </Button>
);

ButtonWithIcon.propTypes = {
  onClick: PropTypes.func.isRequired,
  iconType: PropTypes.string.isRequired,
  icon: PropTypes.string.isRequired,
  label: PropTypes.string.isRequired
};
```

Reply:
@ohadlevy fixed